static/site.css

```diff
@@ -904,23 +916,28 @@ ul.guide li {
   padding: 8px;
   height: var(--site-header-height);
 }
-#btn-settings.shown, #btn-notify.shown {
-  background-color: var(--bg-color1);
+#btn-notify.shown, #notify-app, 
+#btn-settings.shown, #settings {
+  background-color: var(--drop-down);
 }
 #settings, #notify-app {
   display: none;
   flex-direction: column;
   color: var(--font-color);
-  background-color: var(--bg-color1);
-  box-shadow: var(--base-shadow);
+  background-color: var(--drop-down);
+  box-shadow: -1px 5px 6px rgba(0, 0, 0, .3);
   position: absolute;
   top: var(--site-header-height);
   right: 0;
+  border-radius: 5px 0px 0px 5px;
```

Comment:
I have changed the color, shadow, and border-radius of the notifications because these are currently differences between Pychess and Lichess's implementations.

Comparison:

Pychess now 
<img width="400" alt="" src="https://github.com/gbtami/pychess-variants/assets/126312812/f83e3f93-0ba7-45c4-bbdd-c8c2c7c0e046">
<img width="400" alt="" src="https://github.com/gbtami/pychess-variants/assets/126312812/2db8c10b-a8c0-417d-8f0d-36870cf171e8">
After proposal: 
<img width="400" alt="" src="https://github.com/gbtami/pychess-variants/assets/126312812/835e3ab1-8dd5-442d-a74b-c86d092df6ee">
<img width="400" alt="" src="https://github.com/gbtami/pychess-variants/assets/126312812/cebae44b-f48b-49b7-b229-b270b256da3f">
Lichess
<img width="400" alt="" src="https://github.com/gbtami/pychess-variants/assets/126312812/e16c0162-0740-47e3-96ed-efdf23c771b1">
<img width="400" alt="" src="https://github.com/gbtami/pychess-variants/assets/126312812/21397da6-9a4f-41ee-ae52-05ef84065259">